### DISCUSSION TITLE
[台服]自動編隊無法識別

### DISCUSSION THREAD

**Discussion by SaiKamijou** [MaaAssistantArknights/MaaAssistantArknights#7551] [Q&A]
活動關自動戰鬥開始跳自動編隊畫面後不會選擇幹員,接著跳出戰鬥出錯,請問是缺少什麼設置嗎?
![290097620-9860378e-b592-4c27-bb8b-27e0dc4ec393](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/153709211/c200417b-935b-4dfd-ae47-084001a097c2)

這是debug資料夾文件
[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13657392/asst.log)
[gui.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13657394/gui.log)

**Comment by zzyyyl** [comment]
是在游戏画面有 “开始行动” 按钮的时候点的 `开始` 吗，不是的话会出错

**Reply by SaiKamijou** [reply]
有的
就是腳本開始後點編隊時不會自動滑動選取幹員,然後就報錯了。

**Reply by zzyyyl** [reply]
是不是网很卡？

**Reply by zzyyyl** [reply]
去 [issue](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose) 发 [bug 帖](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new?assignees=&labels=bug&projects=&template=cn-bug-report.yaml) 吧，顺便打包传一下 `debug\other` 的截图
